routes/api/contacts.js

```diff
@@ -1,25 +1,78 @@
-const express = require('express')
+const express = require("express");
+const Joi = require("joi");
+const contacts = require("../../models/contacts");
+const { HttpError } = require("../../helpers/index");
+const router = express.Router();
 
-const router = express.Router()
+const addSchema = Joi.object({
+  name: Joi.string().required(),
```

Comment:
please add validation this joi schema only validate if input is made of strings. Email and phone should be validated in another way